### PR TITLE
Refactoring git hooks and linting yaml file

### PR DESCRIPTION
**Pull request by ViBiOh**
# Before creating my pull-request, I ensure:

- [x] Documentation is written
- [x] Unit tests are written and green
- [x] I've cleaned my WIP or meaningless commits
- [x] Examples are up-to-date

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,23 @@
+---
+
 language: go
 go:
-- 1.12.x
-- master
+  - 1.12.x
```
Comment (bdronneau) [inline .travis.yml]: `1.13.X` ?

File: .gitignore [hunk 2]
```diff
@@ -2,5 +2,3 @@ bin/
 release/
 coverage.txt
 dist/
-examples/terraform/.terraform
```
Comment (bdronneau) [inline .gitignore]: I prefer to keep one gitignore

File: .editorconfig [hunk 3]
```diff
@@ -10,3 +11,6 @@ indent_size = 2
 [{Makefile,go.mod,go.sum,*.go}]
 indent_style = tab
 indent_size = 4
+
+[*.py]
```
Comment (bdronneau) [inline .editorconfig]: Nope

File: .travis.yml (hunk 1 above)
Comment (ViBiOh) [reply .travis.yml]: Nop, terraform is not compatible with 1.13

File: .gitignore (hunk 2 above)
Comment (ViBiOh) [reply .gitignore]: Done in b68269c

File: .editorconfig (hunk 3 above)
Comment (ViBiOh) [reply .editorconfig]: Done in b68269c

File: .travis.yml (hunk 1 above)
Comment (bdronneau) [reply .travis.yml]: ok

**Pull request merged**